pkg/controller/manager/manager_controller.go

```diff
@@ -275,41 +275,37 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 
 	// If the manager TLS secret exists, check whether it is managed by the
 	// operator.
-	certOperatorManaged := true
-	if tlsSecret != nil {
-		certOperatorManaged, err = utils.IsCertOperatorIssued(tlsSecret.Data[render.ManagerInternalSecretCertName])
+	var operatorManagedCertSecret bool
+	if installation.CertificateManagement == nil {
+		// If the secret does not exist, then create one.
+		// If the secret exists but is operator managed, then check that it has the
+		// expected DNS names and update it if necessary.
```

Comment:
These comments made more sense before this change. Please update them to better match what is being done here.
Maybe something like
```
We use EnsureCertificateSecret to ensure a secret exists, creating one if one is not passed in. It also ensures the secret passed has the proper DNS names if the secret is operator managed.
```
WDYT? I just think the current comments focus on what the code did so needs a little updating.

Reply:
Makes sense since EnsureCertificateSecret is doing all that work now